pkg/oceancd/model/operator/install.go

```diff
@@ -0,0 +1,171 @@
+package operator
+
+import (
+	"encoding/json"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	"spot-oceancd-operator-commons/component_configs"
+)
+
+type InstallationConfig struct {
+	OceanCDConfig      OceanCDConfig                        `json:"oceancd"`
+	ArgoRolloutsConfig component_configs.ArgoRolloutsConfig `json:"argo"`
+}
+
+func (c *InstallationConfig) GetData() map[string]interface{} {
+	bytes, _ := json.Marshal(c)
+
+	data := map[string]interface{}{}
+	_ = json.Unmarshal(bytes, &data)
+
+	return data
+}
+
+func (c *InstallationConfig) GetOperatorManagerConfig() *component_configs.OperatorManagerConfig {
```

Comment:
No one is using this method, please remove

Reply:
it is used by `buildOperatorManagerConfigMap` method